project/Dependencies.scala

```diff
@@ -76,5 +78,7 @@ object Dependencies {
     val pureconfig       = "com.github.pureconfig"     %% "pureconfig"                % V.pureconfig
     // Scala (test only)
     val specs2           = "org.specs2"                %% "specs2-core"               % V.specs2           % "test"
+    val scalactic        = "org.scalactic"             %% "scalactic"                 % V.scalactic
```

Comment:
Are these two dependencies used only in the tests? If so, you can add `% test` to the end of the line (as for specs2 above) so they don't get bundled into the fat jar when you assemble the release version.